src/js/IAMClient.ts

```diff
@@ -1,30 +1,27 @@
-import type { AppState, AssumeRoleParams, OIDCState } from '../types/state';
+import type { AppState } from '../types/state';
 import type { Credentials } from '../types/zenko';
 import IAM from 'aws-sdk/clients/iam';
-import type { IAMClient as IAMClientInterface, WebIdentityRoles } from '../types/iam';
+import type {
+  IAMClient as IAMClientInterface,
+  WebIdentityRoles,
+} from '../types/iam';
 import { getClients } from '../react/utils/actions';
 import { notFalsyTypeGuard } from '../types/typeGuards';
 export const rolePathName = 'scality-internal/storage-manager-role';
```

Comment:
Is this "default role" still in use by anything ? It should probably not be the case

Reply:
Indeed, got rid of of this default hard-coded rolePathName in https://github.com/scality/zenko-ui/pull/334/commits/52f3f12f9ab44ab7a39d16f97b9ba045c383eacf